build/crd-samples/router/create-rule-eventbus-rest.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rules.kubeedge.io/v1
+kind: Rule
+metadata:
+  name: my-rule-eventbus-rest
+  labels:
+    description: test
+spec:
+  source: "my-eventbus"
+  sourceResource: {"topic":"test","node_name":"xxx"}
+  target: "my-rest"
+  targetResource: {"resource":"http://a.com"}
```

Comment:
Add docs about how to request resource with DNS.

Reply:
It will send the http request with the resoure url. Please refer [here](https://github.com/kubeedge/kubeedge/blob/f3d5941705097ece6863acfb4766f2c05c60f1ae/docs/proposals/cloud-edge-custom-message-design.md#how-to-deliver-custom-messages)

Reply:
In fact, user docs should be split from the design proposal.

So add user docs in kubeedge/website.